app/src/main/java/ch/epfl/sdp/blindly/settings/SettingsLocation.kt

```diff
@@ -46,7 +46,7 @@ class SettingsLocation : AppCompatActivity(), OnMapReadyCallback {
         val loc: LatLng = if (location != null) {
             LatLng(location!!.latitude, location!!.longitude)
         } else {
-            LAUSANNE
+            LAUSANNE_LATLNG

```

Comment:
I think as Tibor said that we don't need the LAUSANNE_LATLING, even more so if it's only used as a default value, maybe it would be better to have an empty list then?